__init__.py

```diff
@@ -163,6 +143,20 @@ def initialize(self):
         # Support query for active alarms from other skills
         self.add_event("private.mycroftai.has_alarm", self.on_has_alarm)
 
+        # establish local timezone from config
+        self.local_tz = self.location_timezone
+        self.log.info("Local timezone configured for %s" % (self.local_tz,))
+
+        self.days = [
+                'monday', 
+                'tuesday', 
+                'wednesday', 
+                'thursday', 
+                'friday', 
+                'saturday', 
+                'sunday'
+                ]
```

Comment:
Here's a possible approach for localized weekday and month names proposed by one of our community members in the [Weather Skill](https://github.com/MycroftAI/skill-weather/pull/156/files):
```Python
from mycroft.util.format import date_time_format

...
    def initialize(self):
        date_time_format.cache(self.lang)
        if self.lang in date_time_format.lang_config.keys():
            self.weekdays = list(date_time_format.lang_config[self.lang]['weekday'].values())
            self.months = list(date_time_format.lang_config[self.lang]['month'].values())
```

Reply:
Thanks Gez, seems to work well.

Reply:
Possible TODO for simplification when MycroftAI/lingua-franca#185 is merged and released (which is currently slated for LF 0.5.0)

That PR doesn't currently expose formatting information via the config interface, because those values live in different resource files. However, it could be rolled into the same accessor, or a separate one for the purpose.

I just reckon something like `list(lingua_franca.config.get('weekdays', self.lang))` would be less painful =P

Obviously, you shouldn't have to work around the parser at all, but the config PR will almost certainly beat the datetime rewrite to the presses.